services/DevHome.Services.WindowsPackageManager/Services/Operations/WinGetInstallOperation.cs

```diff
@@ -32,12 +32,19 @@ public WinGetInstallOperation(
     }
 
     /// <inheritdoc />
-    public async Task<IWinGetInstallPackageResult> InstallPackageAsync(WinGetPackageUri packageUri, Guid activityId)
+    public IAsyncOperationWithProgress<IWinGetInstallPackageResult, WinGetInstallPackageProgress> InstallPackageAsync(WinGetPackageUri packageUri, Guid activityId)

```

Comment:
Question, should all the methods that return a `AsyncInfo.Run` also contain a cancelation token that can be passed into the `AsyncInfo.Run` call? This way the `IAsyncOperation` can be canceled from the UI in the future without much work. Assuming the out of proc installation calls to the WinGet server also handle cancelations as well I guess.